docs: add oasis cli homebrew setup info (#585)
oasisprotocol/cli: docs/setup.mdx

<Tabs>
  <TabItem value="Linux" label="Linux">

>     <Tabs>

Reviewer: I'd avoid 2-level Tab widget. Add Homebrew and Manual as (sub)chapters.

      <TabItem value="Homebrew" label="Homebrew">
        If you use [Homebrew on Linux](https://docs.brew.sh/Homebrew-on-Linux),
        you can install the Oasis CLI with:

>         #### Installation
>         ```shell

Reviewer: Add empty line after any heading. Also below.

```suggestion
        #### Installation
        
        ```shell
```

## Update

> The Oasis CLI includes a built-in command to upgrade to the latest version
> since v0.15.0. To update, run `oasis update`.
> 
> This will check for a newer version on GitHub, show you the release notes,

Reviewer: The version is redundant IMO. But I would mention that this method is suitable for manual installations only.

```suggestion
If you installed Oasis CLI manually, the application includes a built-in
`oasis update` command which upgrades software to the latest version.

This command will check for a newer version on GitHub, show you the release notes,
```

This will check for a newer version on GitHub, show you the release notes,
and ask for confirmation before downloading and replacing the current binary.

> :::info Homebrew users
> If you installed the CLI via Homebrew, you should use `brew upgrade oasis`
> to update it.
> :::

Reviewer: Admonitions should have empty lines in the beginning and end.

```suggestion
:::info Homebrew users

If you installed the CLI via Homebrew, you should use `brew upgrade oasis`
to update it.

:::
```
